samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/storage.c

```diff
@@ -0,0 +1,166 @@
+/* Bluetooth: Mesh Generic OnOff, Generic Level, Lighting & Vendor Models
+ *
+ * Copyright (c) 2018 Vikrant More
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "ble_mesh.h"
+#include "device_composition.h"
+#include "storage.h"
+
+static u8_t STORAGE_ID;
+u8_t reset_counter;
+
+static void save_reset_counter(void)
+{
+	char buf[4];
+
+	settings_str_from_value(SETTINGS_INT8, &reset_counter, buf,
```

Comment:
Please avoid using any other value types than the "bytes". INT8 etc will be removed in the future. So use settings_str_from_bytes() and settings_bytes_from_str() helpers instead.